pkg/apis/serving/v1beta1/component.go

```diff
@@ -73,6 +76,12 @@ type ComponentExtensionSpec struct {
 	// Maximum number of replicas for autoscaling.
 	// +optional
 	MaxReplicas int `json:"maxReplicas,omitempty"`
+	// Replicas specifies the number of replicas that should be deployed. This field is only
+	// relevant when using RawDeployment mode, and it sets a hard replica count.
+	// If you are using an autoscaler, must leave this field unset and
+	// use the MinReplicas and MaxReplicas fields to set the replicas range.
+	// +optional
+	Replicas *int32 `json:"replicas,omitempty"`
```

Comment:
I am not sure if we want to add a new field for the non autoscaler case, we can still use the minReplica and it means it never scales up, also user does not need to change to use a different field if decides to update to add the autoscaler.